pkg/recovery/etcd.go

```diff
@@ -10,9 +10,8 @@ import (
 	"path"
 	"strings"
 
-	"github.com/kubernetes-sigs/bootkube/pkg/asset"
-
 	"github.com/coreos/etcd/clientv3"
+	"github.com/kubernetes-sigs/bootkube/cmd/render/plugin/default/asset"
```

Comment:
I think this import separation is fine, this is what `goimports` enforces, to keep imports within the module separate from external dependencies.

Usually you have 3 groups: stdlib, external, internal, in this order.

Reply:
I usually have `gofumports` to handle that ;) . Will fix 👍.